### DISCUSSION TITLE
windows 安装插件需要重复好几次

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#70] [Q&A]
windows wsl 安装插件，git 可以正常访问，部分插件也会安装成功。
但是每次安装都会失败很多插件
![image](https://user-images.githubusercontent.com/77786952/182063239-f3ed477a-d656-48bf-b107-e5a772415f2d.png)
这个时候就需要不停 PackerSync 重复好几次... 蛮离谱的
有办法优化么？
![image](https://user-images.githubusercontent.com/77786952/182063288-5899ca89-b73e-4fde-9ff0-e264aca3243b.png)
感觉是不是一次安装太多了，阻塞

**Comment by nshen** [comment]
plugins.lua 里我记得是有设置并发数量的，除了挂梯子换个好点的线路，就是用github镜像，也没什么好办法了
